bchain/coins/eth/evm.go

```diff
@@ -92,6 +92,11 @@ type EthereumNewBlock struct {
 	channel chan *types.Header
 }
 
+// NewEthereumNewBlock returns an initialized EthereumNewBlock struct
+func NewEthereumNewBlock() *EthereumNewBlock {
```

Comment:
add constructor function to keep channel private if able to leverage base ethereum evm types for other evm chains